spec/spec_helper.rb

```diff
@@ -18,7 +18,3 @@ def input(plugin, &block)
   end
 
 end
-
-RSpec.configure do |c|
-  c.include IrcHelpers
```

Comment:
will open an issue for this in the logstash-core repo, I agree is also a
problem with what you mention.

thanks

On Wed, Oct 7, 2015 at 4:33 PM João Duarte <notifications@github.com> wrote:

> In spec/spec_helper.rb
> <https://github.com/logstash-plugins/logstash-input-irc/pull/17#discussion_r41397206>
> :
>
> > @@ -18,7 +18,3 @@ def input(plugin, &block)
> >    end
> >
> >  end
> > -
> > -RSpec.configure do |c|
> > -  c.include IrcHelpers
>
> +1
> we need to find a way to namespace the includes. this is not just a
> devutils/plugin problem, but 2 plugins doing a conflicting include will
> also cause problems.
> thanks for the input, I'm going to create the prs for the others
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/logstash-plugins/logstash-input-irc/pull/17/files#r41397206>
> .
>